spec/models/doorkeeper/access_token_spec.rb

```diff
@@ -131,9 +131,9 @@ module NoGenerate
         token1 = FactoryGirl.create :access_token, use_refresh_token: true
         token2 = FactoryGirl.create :access_token, use_refresh_token: true
         expect do
-          token2.write_attribute :refresh_token, token1.refresh_token
+          token2.send(:write_attribute, :refresh_token, token1.refresh_token)
```

Comment:
`write_attribute` is private method. And `raise_error` caught this exception instead of `ActiveRecord::RecordNotUnique`